### DISCUSSION TITLE
Requesting 2.0.6 for Issue 924 (ConcurrentModificationException fix)

### DISCUSSION THREAD

**Discussion by TIBCOeddie** [eclipse-m2e/m2e-core#999] [General]
Hello!

Our product uses m2e extensively, and our users see ConcurrentModificationException a fair amount - thankfully this might be history now with https://github.com/eclipse-m2e/m2e-core/issues/924 ... although I can't tell when it might be released!

**Comment by mickaelistria** [comment]
We'll first wait for a release of Wild Web Developer that uses newer LemMinX (next release is planned early November) and then update m2e to use newer LemMinX-Maven that will provide several pom edition enhancement.

**Comment by TIBCOeddie** [comment]
Thoughts on considering 2.0.6 for a bug fix of visible impact (CME either gets into error logs or worse, shown to the user unexpectedly)?
LemMinX pom editor sounds great! ... for 2.1?

**Reply by HannesWell** [reply]
Can't you use the https://download.eclipse.org/technology/m2e/snapshots/2.1.0/ repo? It will vanish after the 2.1.0 release so you or your customers will only use m2e snapshots until that release.
If you want to have a fixed repo you could mirror the current m2e snapshot repo and provide the mirror to your customers until 2.1 is released.

**Comment by TIBCOeddie** [comment]
As you can imagine, we would not be shipping snapshots in our releases...

**Comment by HannesWell** [comment]
I'm not sure of the other committer's intend, but I had the intend to release M2E 2.1.0 in time for the December Eclipse Simultaneous Release in about a month.
At the moment there are multiple developments that I would like to have in the that release so I would prefer to not release much erlier.

Nevertheless I just created the following `snapshots/2.1.0-M1` repo as copy of the `snapshots/2.1.0` that will not be changed anymore (so you can test its content). That repo will exist until the 2.1.0 release and if that helps you we can keep it a a few weeks/a month longer so you can switch to 2.1.0. Also note that a non existing repo is usually not a problem, since p2 simply ignores it.

- https://download.eclipse.org/technology/m2e/snapshots/2.1.0-M1/

@TIBCOeddie, does this help you?

**Comment by TIBCOeddie** [comment]
It will help for testing, of course. We just wouldn't/couldn't ship it.
Bummer no 2.0.6; but the Dec simrel will be an okay compromise. If we use
2.1.0-M1 it would be now, and we won't need it for longer than upon release.

**Comment by HannesWell** [comment]
@TIBCOeddie M2E 2.1.0 has been released (https://github.com/eclipse-m2e/m2e-core/pull/1085).

Please let me know when you are not using the 2.1.0 milestone repos anymore.

**Reply by kjsmita6** [reply]
We were never using it so if you are wondering to delete it go ahead :)

**Reply by HannesWell** [reply]
Alright. Thanks for letting me know. 👍🏽
